_posts/0000-01-02-kushalbhoopalam007.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
kushal bhoopalam

Reply:
# The largest heading
## The second largest heading
###### The smallest heading